tests/acl/test_stress_acl.py

```diff
@@ -91,12 +91,15 @@ def prepare_test_file(rand_selected_dut):
 @pytest.fixture(scope='module')
 def prepare_test_port(rand_selected_dut, tbinfo):
     mg_facts = rand_selected_dut.get_extended_minigraph_facts(tbinfo)
-    if tbinfo["topo"]["type"] == "mx":
-        dut_port = mg_facts["minigraph_acls"]["DataAcl"][0]
-    else:
-        dut_port = list(mg_facts['minigraph_portchannels'].keys())[0]
+
+    ports = list(mg_facts['minigraph_portchannels'])
```

Comment:
hi @justin-wong-ce I had a discussion with Xiawei, quote from his response
"""
The purpose is to bind acl to a upstream port, both pc or eth should work, I remember when I write the case, some tb do not have data acl table, so I used pc, but I checked again this afternoon and found t0/t1/dualtor/mx all have data acl table
"""
the port selection logic could be, take pc first, if no, check the dataacl ports. But there should be no difference for dualtor, could we remove dualtor condition check from line 96, it is a little confusing.